Code/Framework/AzToolsFramework/AzToolsFramework/UI/Outliner/EntityOutliner.qss

```diff
@@ -21,15 +21,15 @@ AzToolsFramework--EntityOutlinerWidget QTreeView
 AzToolsFramework--EntityOutlinerWidget QTreeView::branch:hover
 , AzToolsFramework--EntityOutlinerWidget QTreeView::item:hover
 {
-    background: rgba(255, 255, 255, 30);
+    background: transparent;
```

Comment:
Maybe throw in a comment here that the rendering for hover now lives in `EntityOutlinerTreeView::PaintBranchSelectionHoverRect`?

Reply:
Yes will do!